### PR TITLE
Update page with PATCH

### PR DESCRIPTION
**Pull request by osahyoun**

### CODE REVIEW

**Comment by paulaferris** [comment]
@osahyoun: Just for our records and sanity in the future, I'd love if you could provide a brief description on pull requests explaining what they are doing and why :)

File: spec/lib/rest/page_spec.rb [hunk 1]
```diff
@@ -73,9 +73,8 @@
         data[:id] = "99999999999" # No page with this ID exists
       end
 
-      it 'returns 400' do
```
Comment (osahyoun) [inline spec/lib/rest/page_spec.rb:76]: AK have fixed this. A page not found now correctly returns a 404.

**Comment by osahyoun** [comment]
`PUT`s now make a full update on page records, so switching to `PATCH`, for partial updates.

**Comment by osahyoun** [comment]
`PUT` now makes a full update on page records. This PR switches to `PATCH`to allow partial updates.

**Pull request merged**